parity/configuration.rs

```diff
@@ -73,6 +73,7 @@ impl Configuration {
 	pub fn into_command(self) -> Result<Cmd, String> {
 		let dirs = self.directories();
 		let pruning = try!(self.args.flag_pruning.parse());
+		let pruning_history = self.args.flag_pruning_history;
```

Comment:
History bellow a certain amount shouldn't be allowed until we have a way to recover from a prehistoric reorganization.